react-components/src/notifications/model/notificationCategory.js

```diff
@@ -0,0 +1,11 @@
+const notificationCategory = {
+    all: "All",
+    new: "New",
+    analysis: "Analysis",
+    data: "Data",
+    tool_request: "Tool Request",
+    apps: "Apps",
+    permanent_id_request: "Permanent ID Request",
+    team: "Team",
+};
```

Comment:
Not necessary for this PR, but I'm wondering if there's some way we can share models between GWT and React so it doesn't have to be duplicated 🤔